src/app/global.css

```
@@ -124,21 +168,57 @@
     @apply py-8 sm:py-12 lg:py-16;
   }
 
+  /* 基本のガラス面（ナビ・パネル用） */
   .glass {
     @apply rounded-[var(--radius-m)]
            backdrop-blur-[var(--blur-m)]
            bg-[color:var(--glass-surface)]
-           border border-[color:var(--glass-border)]
-           shadow-[0_6px_26px_rgba(0,0,0,.12)];
+           border border-[color:var(--glass-border)];
+    box-shadow: var(--card-shadow);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)